module-system/module-implementations/sov-evm/src/tests/mod.rs

```diff
@@ -0,0 +1,2 @@
+mod call_tests;
```

Comment:
Why won't place tests as integration tests in `sov-evm/tests` ? Then it will be covering public API.

Reply:
We are still accessing private fields, the tests will be moved after we have a convenient rpc methods for accessing the module state